### PR TITLE
cJSON debian package

### PR DESCRIPTION
**Pull request by sachinholla**
**- Why I did it**
An upcoming enhancement to mgmt-framework CLI requires opensource cJOSN
library for parsing JSON data. Adding as a deb package makes it easy to
use in other components too.

**- How I did it**
Added build scripts to download and package cJSON v1.7.11 as a
debian package - libcjson_1.7.11_amd64.deb. Components that need C/C++
JSON parser can use this tool.

**- How to verify it**
Full build

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

cJSON debian package

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/libcjson/Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = $(LIBCJSON)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Obtain libcjson
+	rm -rf ./cJSON-$(LIBCJSON_VERSION)
+	wget -N "https://github.com/DaveGamble/cJSON/archive/v$(LIBCJSON_VERSION).tar.gz"
```
Comment (lguohan) [inline src/libcjson/Makefile:10]: @xumia, does this align with reproduciable build design?

**Comment by sachinholla** [comment]
retest vs please

**Comment by sachinholla** [comment]
retest vsimage please

**Comment by sachinholla** [comment]
retest vsimage please

**Pull request closed without merging**